products/ansible.md

```diff
@@ -16,8 +16,14 @@ purls:
 auto:
 -   git: https://github.com/ansible-community/ansible-build-data.git
 releases:
--   releaseCycle: "6"
+-   releaseCycle: "7"
     eol: false
+    latest: "7.0.0"
+    latestReleaseDate: 2022-11-22
+    releaseDate: 2022-11-22
+
+-   releaseCycle: "6"
+    eol: 2022-11-22
```

Comment:
> Unmaintained (end of life) after Ansible 6.7.0

Ref: https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-community-changelogs

This should be supported for another 3 weeks or so (minor release cadence), so lets use R+4 weeks instead to be conservative.

Reply:
Sure, will use 2022-12-22.